portable/ThirdParty/GCC/Posix/port.c

```diff
@@ -461,6 +461,8 @@ static void * prvWaitForStart( void * pvParams )
     uxCriticalNesting = 0;
     vPortEnableInterrupts();
 
+    pthread_setname_np(pxThread->pthread, pcTaskGetName(NULL));
```

Comment:
We do not allow *_np (non portable functions), as they would break some  Posix builds (mac os), that are not GNU specific systems
maybe we can surround it with some ifdefs / endif to be omitted on non gnu systems

Reply:
I have removed that change from this PR. I'll probably include it in another PR once I have found a portable way of implementing it.